Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -7,19 +12,30 @@ public static class ArrayOperations
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
-        public static void Sort(int[] array)
+        public static int[] Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null)
+            {
+                return array;
+            }
+            else
+            {
+                Array.Sort(array);
+                return array;
+            }
         }
 
         /// <summary>
         /// Reverse the array elements, first being last and so on.
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
-        public static void Reverse(int[] array)
+        public static int[] Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null)
+                return array;
+            Array.Reverse(array);
```

Comment:
Don't cheat! 😄 Please implement this by yourself. No C#-unique language features are yet allowed.